libvirt/uri/connection_uri.go

```diff
@@ -31,17 +31,16 @@ func Parse(uriStr string) (*ConnectionURI, error) {
 // The name passed to the remote virConnectOpen function is formed by removing
 // transport, hostname, port number, username and extra parameters from the remote URI
 // unless the name option is specified.
-func (u *ConnectionURI) RemoteName() string {
+func (u ConnectionURI) RemoteName() string {
```

Comment:
This is better style but ultimately does nothing, since the copied `u` value still contains a pointer to the same `url.URL` value.